Userland/Libraries/LibGfx/Color.cpp

```diff
@@ -63,7 +64,12 @@ static Optional<Color> parse_rgba_color(StringView string)
     auto g = parts[1].to_int().value_or(256);
     auto b = parts[2].to_int().value_or(256);
 
-    double alpha = strtod(parts[3].to_string().characters(), nullptr);
+    double alpha = 0;
+    char const* start = parts[3].characters_without_null_termination();
+    auto alpha_result = parse_first_floating_point(start, start + parts[3].length());
```

Comment:
wait, is it intentional that `Color::from_string("rgba(banana, apple, pomegranate, pineapple)"sv)` is a value of `Color(256,256,256,0)`?

Reply:
I don't know, but that's what the code does already :shrug:

Reply:
The actual CSS parser has its own implementation of rgba which doesn't have this behaviour. However Color::from_string which can call this method is used in a bunch of other spots unguarded so this is probably incorrect.